src/assets/discrete-assets.md

```diff
@@ -77,3 +77,73 @@ curl -X POST "https://service.centrapay.com/api/external-assets" \
 | Status | Code                    | Description                                         |
 |:-------|:------------------------|:----------------------------------------------------|
 | 403    | {% break _ FORBIDDEN %} | Invalid external id or pin or asset already exists. |
+
+## Get Asset
```

Comment:
These two endpoints are not specific to discrete assets or wallets so I am not sure where to put them. What about the parent page?